### PR TITLE
Harshada assignment 22

### PR DESCRIPTION
**Pull request by HarshadaJagtap11**

### CODE REVIEW

File: src/harshada/MaxFrequency.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+		int maxFreq= 0 ;
+		char maxchar= ' ' ;
+		for( int index=0; index<input.length(); index++) {
+		char ab = input.charAt(index);
+		int freq = getCharfreq( input , ab);
+			if( maxFreq <= freq ) {
+			maxFreq = freq ;
+			maxchar = ab ;
+			}
+		}System.out.println(input+" -> "+maxchar+" -> "+maxFreq);
```
Comment (KrishnaKTechnocredits) [inline src/harshada/MaxFrequency.java:30]: Sop should be in the next line.

**Review by KrishnaKTechnocredits — CHANGES_REQUESTED** [review]
Please work on indentation and change code as requested.

File: src/harshada/MaxFrequency.java (hunk 1 above)
Comment (HarshadaJagtap11) [reply src/harshada/MaxFrequency.java]: Code has been changed as requested

**Comment by HarshadaJagtap11** [comment]
> Please work on indentation and change code as requested.

Indentation has been corrected

**Comment by HarshadaJagtap11** [comment]
Output of Code 1:
image
Output of Code 2:
image

**Review by MargiVegada — APPROVED** [review]
Code looks good to me

**Pull request merged**